pages/posts/[slug].js

```diff
@@ -9,33 +27,29 @@ export default function Home(props) {
             backgroundColor: "lightgray",
           }}
         >
-          {JSON.stringify(props.data.getPostDocument.data, null, 2)}
+          {JSON.stringify(data.getPostDocument.data, null, 2)}
         </pre>
       </code>
     </Layout>
   );
 }
 
 export const getStaticPaths = async () => {
-  const query = gql`
-    {
-      getPostList {
-        edges {
-          node {
-            sys {
-              filename
+  const tinaProps = await staticRequest({
```

Comment:
Since we not really use this as `props` would a name like `res` or `cmsData` or `tinaData` be better?